### PR TITLE
RQ related conveniences

### PR DESCRIPTION
**Pull request by gsong**
* Add [rq-dashboard](https://github.com/ducu/rq-dashboard) to our tool belt
* Add convenience make tasks for working with rq related tools

**Note**: Include `base.in` in `dev.in` so we don't get conflicts in Flask versions.

/cc @edx/ospr

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -22,12 +22,38 @@ pip-compile: ## update the requirements/*.txt files with the latest packages sat
...
+	done
+	open http://localhost:9181
+
+
+rq-requeue-failed:
+	@$(MAKE) cmd="rq requeue -a" rq-cmd
+
+
+rqinfo:
+	@$(MAKE) cmd=rqinfo rq-cmd
```
Comment (nedbat) [inline Makefile]: I don't understand all the make-magic here.  There really wasn't a simpler way to achieve what we need?

**Comment by edx-webhook** [comment]
Testing 123…

**Comment by edx-webhook** [comment]
Won't you please take the survey?

**Test of Markdown**

<https://google.com>

**Comment by edx-webhook** [comment]
Even though your PR wasn't merged, please take a moment to [answer a two
question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1813723273&entry.1671973413=edx/openedx-webhooks&entry.867055334=https://github.com/edx/openedx-webhooks/pull/96&entry.1484655318=https://github.com/gsong&entry.752974735=2016-11-12&entry.1917517419=2017-08-14&entry.2133058324=No) so we can improve your experience.

**Comment by edx-webhook** [comment]
@gsong Even though your pull request wasn't merged, please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1813723273&entry.1671973413=edx/openedx-webhooks&entry.867055334=https://github.com/edx/openedx-webhooks/pull/96&entry.1484655318=https://github.com/gsong&entry.752974735=2016-11-12&entry.1917517419=2017-08-14&entry.2133058324=No) so we can improve your experience.

**Comment by edx-webhook** [comment]
@gsong 🎉 Your pull request was merged!

Take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1813723273&entry.1671973413=edx/openedx-webhooks&entry.867055334=https://github.com/edx/openedx-webhooks/pull/96&entry.1484655318=https://github.com/gsong&entry.752974735=2016-11-12&entry.1917517419=2017-08-14&entry.2133058324=No) so we can improve your experience.

**Comment by edx-webhook** [comment]
@gsong 🎉 Your pull request was merged!

Take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1813723273&entry.1671973413=edx/openedx-webhooks&entry.867055334=https://github.com/edx/openedx-webhooks/pull/96&entry.1484655318=https://github.com/gsong&entry.752974735=2016-11-12+17:04&entry.1917517419=2017-08-14+19:42&entry.2133058324=No) so we can improve your experience.

**Comment by edx-webhook** [comment]
@gsong 🎉 Your pull request was merged!

Take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1813723273&entry.1671973413=edx/openedx-webhooks&entry.867055334=https://github.com/edx/openedx-webhooks/pull/96&entry.1484655318=https://github.com/gsong&entry.752974735=2016-11-12+17:04&entry.1917517419=2017-08-14+19:42&entry.2133058324=No) so we can improve your experience in the future.

**Comment by edx-webhook** [comment]
@gsong Even though your pull request wasn't merged, please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1813723273&entry.1671973413=edx/openedx-webhooks&entry.867055334=https://github.com/edx/openedx-webhooks/pull/96&entry.1484655318=https://github.com/gsong&entry.752974735=2016-11-12+17:04&entry.1917517419=2017-08-14+19:42&entry.2133058324=No) so we can improve your experience in the future.

**Comment by edx-webhook** [comment]
@gsong Even though your pull request wasn’t merged, please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1813723273&entry.1671973413=edx/openedx-webhooks&entry.867055334=https://github.com/edx/openedx-webhooks/pull/96&entry.1484655318=https://github.com/gsong&entry.752974735=2016-11-12+17:04&entry.1917517419=2017-08-14+19:42&entry.2133058324=No) so we can improve your experience in the future.

**Comment by edx-webhook** [comment]
@gsong Even though your pull request wasn’t merged, please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1813723273&entry.1671973413=edx/openedx-webhooks&entry.867055334=https://github.com/edx/openedx-webhooks/pull/96&entry.1484655318=https://github.com/gsong&entry.752974735=2016-11-12+17:04&entry.1917517419=2017-08-14+19:42&entry.2133058324=No) so we can improve your experience in the future.

**Comment by edx-webhook** [comment]
@gsong Even though your pull request wasn’t merged, please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1813723273&entry.1671973413=edx/openedx-webhooks&entry.867055334=https://github.com/edx/openedx-webhooks/pull/96&entry.1484655318=https://github.com/gsong&entry.752974735=2016-11-12+17:04&entry.1917517419=2017-08-14+19:42&entry.2133058324=No) so we can improve your experience in the future.

**Comment by edx-webhook** [comment]
@gsong 🎉 Your pull request was merged!

Take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1813723273&entry.1671973413=edx/openedx-webhooks&entry.867055334=https://github.com/edx/openedx-webhooks/pull/96&entry.1484655318=https://github.com/gsong&entry.752974735=2016-11-12+17:04&entry.1917517419=2017-08-14+19:42&entry.2133058324=Yes) so we can improve your experience in the future.

**Comment by edx-webhook** [comment]
@gsong Even though your pull request wasn’t merged, please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1813723273&entry.1671973413=edx/openedx-webhooks&entry.867055334=https://github.com/edx/openedx-webhooks/pull/96&entry.1484655318=https://github.com/gsong&entry.752974735=2016-11-12+17:04&entry.1917517419=2017-08-14+19:42&entry.2133058324=No) so we can improve your experience in the future.

**Comment by edx-webhook** [comment]
@gsong Even though your pull request wasn’t merged, please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1813723273&entry.1671973413=edx/openedx-webhooks&entry.867055334=https://github.com/edx/openedx-webhooks/pull/96&entry.1484655318=https://github.com/gsong&entry.752974735=2016-11-12+17:04&entry.1917517419=2017-08-14+19:42&entry.2133058324=No) so we can improve your experience in the future.

**Comment by edx-webhook** [comment]
@gsong Even though your pull request wasn’t merged, please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1813723273&entry.1671973413=edx/openedx-webhooks&entry.867055334=https://github.com/edx/openedx-webhooks/pull/96&entry.1484655318=https://github.com/gsong&entry.752974735=2016-11-12+17:04&entry.1917517419=2017-08-14+19:42&entry.2133058324=No) so we can improve your experience in the future.

**Comment by edx-webhook** [comment]
@gsong 🎉 Your pull request was merged!

Take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1813723273&entry.1671973413=edx/openedx-webhooks&entry.867055334=https://github.com/edx/openedx-webhooks/pull/96&entry.1484655318=https://github.com/gsong&entry.752974735=2016-11-12+17:04&entry.1917517419=2017-08-14+19:42&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request closed without merging**